features/prefetch.yml

```diff
@@ -0,0 +1,7 @@
+name: Prefetch
+description: The `Sec-Purpose` HTTP header with a value of `prefetch` signals that a resource can be requested in anticipation of a near future navigation, such as a page linked in search results or a link that a user has hovered over.
+spec: https://wicg.github.io/nav-speculation/prefetch.html
+compat_features:
+  - api.PerformanceResourceTiming.deliveryType.navigational-prefetch
+  - http.headers.Sec-Purpose
+  - http.headers.Sec-Purpose.prefetch
```

Comment:
AFAICT, these are request headers sent when making a request from a `<link rel="prefetch">`. I think these keys should go in the `link-rel-prefetch` feature instead.

There won't be anything left in this feature after this, so it can be deleted.